process_manager/tables.py

```diff
@@ -1,61 +1,130 @@
-"""Tables for the process_manager app."""
+"""Tables configuration for the Process Manager data display."""
+
+from typing import ClassVar
 
 import django_tables2 as tables
 from django.utils.safestring import mark_safe
 
 logs_column_template = (
-    "<a href=\"{% url 'process_manager:logs' record.uuid %}\">LOGS</a>"
+    "<a href=\"{% url 'process_manager:logs' record.uuid %}\" "
+    'class="btn btn-sm btn-primary text-white" title="View logs">LOGS</a>'
 )
 
-header_checkbox_hyperscript = "on click set .row-checkbox.checked to my.checked"
-
-row_checkbox_hyperscript = """
-on click
-if <.row-checkbox:not(:checked)/> is empty
-  set #header-checkbox.checked to true
-else
-  set #header-checkbox.checked to false
-"""
```

Comment:
Why was this removed? 
The header checkbox does not work anymore.

Reply:
Sorry, I've lost it one the way. I will add it back